packages/editor/src/components/collab-sidebar/comments.js

```diff
@@ -317,9 +317,11 @@ export function Comments( {
 					setShowCommentBoard={ setShowCommentBoard }
 					commentSidebarRef={ commentSidebarRef }
 				/>
-				<Text as="p">{ __( 'No notes available.' ) }</Text>
+				<Text as="p">{ __( 'Select a block to insert a Note.' ) }</Text>
```

Comment:
```suggestion
				<Text as="p">{ __( 'Select a block to insert a note.' ) }</Text>
```

I think other places have recommended using lowercase n note instead of uppercase N Note.